testsuite/run_forge.sh

```diff
@@ -9,18 +9,32 @@
 # if the necessary environment variables are set.
 #
 
-# ensure the script is run from project root
-pwd | grep -qE 'aptos-core$' || (echo "Please run from aptos-core root directory" && exit 1)
-
-# for calculating regression in local mode
-LOCAL_P99_LATENCY_MS_THRESHOLD=60000
+# Default to off
+# WRAPPER_KILLSWITCH=true
+# REMOVE THIS
+WRAPPER_KILLSWITCH=false
 
 # output files
 FORGE_OUTPUT=${FORGE_OUTPUT:-$(mktemp)}
 FORGE_REPORT=${FORGE_REPORT:-$(mktemp)}
 FORGE_PRE_COMMENT=${FORGE_PRE_COMMENT:-$(mktemp)}
 FORGE_COMMENT=${FORGE_COMMENT:-$(mktemp)}
 
+if [[ $WRAPPER_KILLSWITCH = true ]]; then
```

Comment:
If you expect these to be env vars, you'll need to add it to `run-forge.yaml` as an env that's populated from GH secret